mzLib/MassSpectrometry/PeakIndexing/IndexingEngine.cs

```diff
@@ -121,7 +123,7 @@ public List<IIndexedPeak> GetXic(double mz, double retentionTime, Tolerance ppmT
         /// <param name="missedScansAllowed"> the number of successive missed scans allowed before the xic is terminated </param>
         /// <param name="maxPeakHalfWidth"> the maximum distance from the apex RT of the XIC to both start RT and end RT </param>
         /// <returns> A list of IIndexedPeak objects, ordered by retention time </returns>
-        public List<IIndexedPeak> GetXic(double mz, int zeroBasedStartIndex, Tolerance ppmTolerance, int missedScansAllowed, double maxPeakHalfWidth = double.MaxValue)
+        public List<IIndexedPeak> GetXic(double mz, int zeroBasedStartIndex, Tolerance ppmTolerance, int missedScansAllowed, double maxPeakHalfWidth = double.MaxValue, int? charge = null)
```

Comment:
I would like a summary comment for the new arguments, especially because if it is not null, you won't get any results at all